### ISSUE TITLE
Flag old GLs versions as old and point to their replacement

### ISSUE THREAD

**Issue by hcayless** [TEIC/Stylesheets#470]
Consider injecting some JavaScript to surface a (non-modal) alert noting that older versions of the Guidelines have been superseded and by what.

**Comment by martindholmes** [comment]
@raffazizzi I'm not sure I like the idea of a dialog box popup; how about a CSS rule that could add some content to the footer based on something in the footer like:

`a.link_ref[href~="4.10.1"]:after{
  content: " OLD VERSION";
  background-color: pink;
}`

We could add a new rule to the stylesheet for every release that cycles out of current, and that way we would never actually have to change the pages themselves.

**Comment by martindholmes** [comment]
If we did this with pure CSS, it would be a TEI ticket, and would also affect Documentation (TCW22).

**Comment by raffazizzi** [comment]
I agree. I usually prefer to place this type of warning at the top rather than the footer, though. See https://scholarlyediting.org/se.index.issues.html

**Comment by raffazizzi** [comment]
It's also unlikely that CSS content will be considered by search engines. Perhaps it doesn't matter, but adding an HTML banner to the existing content would address this

**Comment by joeytakeda** [comment]
I do think this would likely be better in JS since it’s not something that needs to be part of the *actual* guidelines, but just as part of the guidelines when they are in the vault and not current. 

I think a banner at the top that’s dismissible, which is created on page load iff the page is loaded from the Vault URL?

**Comment by martindholmes** [comment]
@joeytakeda If we create a banner that requires dismissal, then either it'll have to be dismissed on every page, or we'll have to put a flag in the user's local storage to prevent that. The latter would go against our intention to avoid a GDPR popup (https://github.com/TEIC/TEI/issues/2748). For this reason, I think a less prominent warning would be better.
